src/Components/Form/Form.js

```diff
@@ -1,76 +1,76 @@
-import './Form.css';
-import { useState } from 'react';
+import "./Form.css";
+import { useState } from "react";
+import { gql, useMutation } from "@apollo/client";
+
+const POST_REVIEW = gql`
```

Comment:
I'd recommend moving queries and mutations into their own directories for cleaner and easier to read code.